mod.ts

```diff
@@ -9,22 +9,9 @@ import Database from "./data/postgres";
 export default class Mod {
 
     static async mod(interaction: ChatInputCommandInteraction, db: Database) {
-        if (!interaction.guild?.available) {
-            await interaction.reply({
-                content: 'Guild not available  (score -> score -> interaction.guild.available is either null or false)',
-                ephemeral: true
-            });
-            return;
-        }
-        if (!interaction.guildId) {
-            await interaction.reply({
-                content: 'Internal Server Error (score -> score -> interaction.guildId = null)',
-                ephemeral: true
-            });
-            return;
-        }
+
```

Comment:
Do you think this empty line should be kept?

Reply:
Removed the line